src/Engine/Graphics/Renderer/BaseRenderer.cpp

```diff
@@ -287,21 +287,47 @@ void BaseRenderer::PrepareDecorationsRenderList_ODM() {
                     if (frame->uFlags & 0x40000) v38 |= 0x40;
                     if (frame->uFlags & 0x20000) v38 |= 0x80;
 
-                    // for light
+                    // This is an attempt to make the game look more nice.
+                    // In MM7 data some sprites like a "cauldron" do not emit light.
+                    // In MM8 pedestals and campfires have large radius and bright color (too bright actually).
+                    // The following changes makes OpenEnroth look more like MM8.
+                    // OpenEnroth do not support mods and data patches right now.
+                    // So the changes are in C++ code. But it better be placed in lua scripts or binary data patches.
+                    const int uDecorationDescID = pLevelDecorations[i].uDecorationDescID;
+                    int goodGlowRadius = 512;
+
+                    if (uDecorationDescID == 5) { // campfire, like the one you see when the the game is just started
+                        frame->uGlowRadius = goodGlowRadius;
+                    } else if (uDecorationDescID == 6) { // cauldron
+                        frame->uGlowRadius = goodGlowRadius;
+                    } else if (uDecorationDescID == 27) { // signal fire pit
+                        frame->uGlowRadius = goodGlowRadius;
+                    } else if (uDecorationDescID == 222) { // signal fire pit (unlit)
+                    } else if (uDecorationDescID == 184) { // fire?
+                    } else if (uDecorationDescID == 187) { // fire?
+                    } else if (uDecorationDescID == 190) { // fire?
+                    } else if (uDecorationDescID >= 206 && uDecorationDescID <= 209) { // lighthouse fire
+                        frame->uGlowRadius = goodGlowRadius;
+                    } else if (uDecorationDescID >= 210 && uDecorationDescID <= 221) { // magic pedestal
+                        frame->uGlowRadius = goodGlowRadius;
+                    } else if (uDecorationDescID == 190) {
+                    }
+
+                    // Pure white is too bright for a light source
+                    Color defaultLightColor = Color(185, 185, 185);
```

Comment:
Please can you put this behind a config option.

Reply:
Yeah! But how the config should look?

Something like this:
```
// src/Application/GameConfig.h
    class Graphics : public ConfigSection {
        ....
        ColorEntry DefaultLightColor = {this, "default_light_color", Color(185, 185, 185), "Some light sources have black color. Use this color for such lights instead."};
        Bool AddMoreLights = {this, "add_more_lights", true, "force MM7 pedestals/cauldrons/etc to emit light (as in MM8)"};
        Int DefaultLightRadius = {this, "default_light_radius", 172, "Light radius of the additional lights."};
```

Reply:
Yeah this looks good!